packages/default-theme/components/account/MyAddresses.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="shipping-list">
+      <Address v-for="address in addresses"
+      :address="address"
+      :key="address.id"
+      :isDefaultShipping="address.id === selectedShipping"
+      :isDefaultBilling="address.id === selectedBilling"
+      v-on:selectDefaultAddress="selectDefaultAddress"
```

Comment:
There is a shortcut for `v-on:`, `@selectDefaultAddress`

Reply:
nice :+1: changing it!